packages/edit-site/src/components/block-editor/use-site-editor-settings.js

```diff
@@ -133,7 +133,7 @@ export function useSpecificEditorSettings() {
 	);
 	const archiveLabels = useArchiveLabel( templateSlug );
 	const defaultRenderingMode = postWithTemplate ? 'template-locked' : 'all';
-	const goBack = useGoBack();
+	const onNavigateToPreviousEntityRecord = useGoBack();
```

Comment:
Maybe should rename the `useGoBack` hook?

Reply:
[Done](https://github.com/WordPress/gutenberg/pull/58416/commits/d4b08fc19ed2c4f6dbd61c83823552c7ff82e59f)